static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -331,28 +331,46 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     const hasIntegration = configurations ? configurations.length > 0 : false;
     const endpoint = `/organizations/${organization.slug}/`;
     const hasOrgWrite = organization.access.includes('org:write');
-
+    let openPRDisabledReason = t(
+      'You must have a GitHub integration to enable this feature.'
+    );
+    if (!organization.features.includes('integrations-open-pr-comment')) {
+      openPRDisabledReason = t("This feature isn't available to you yet.");
+    }
```

Comment:
if you hide the toggle when the org doesn't have the feature flag you won't need this extra reason